src/App.tsx

```diff
@@ -8,52 +8,38 @@ import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
-import { getUserPosts } from './api/posts';
-import { User } from './types/User';
-import { Post } from './types/Post';
-import { Counter } from './features/counter/Counter';
+// import { getUserPosts } from './api/posts';
```

Comment:
Remove all comments. Try it write obvious code that explain all we need without comments)